src/features/retrospective/ui/RetrospectiveSummary.tsx

```
@@ -0,0 +1,94 @@
+          />
+        ))}
+      </div>
+      <ul className='flex flex-col gap-6'>
+        {summaries.map(({ color, title, percentage, amount }) => (
+          <SummaryItem
+            color={color}
+            title={title}
+            percentage={percentage}
+            amount={amount}
+          />
+        ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)